Scripts/script-DemistoLeaveInvestigation.yml

```diff
@@ -0,0 +1,62 @@
+commonfields:
+  id: DemistoLeaveInvestigation
+  version: -1
+name: DemistoLeaveInvestigation
+script: |-
+  var body = {};
+
+  if (args.InvestigationID) {
+      var invID = args.InvestigationID;
+      var res = executeCommand("getIncidents", {id: invID});
+      if (isError(res[0])) {
+          return res[0].Contents;
+      }
+      var owner = res[0].Contents.data[0].owner;
+
+  } else {
+      var invID = incidents[0].id;
+      var owner = incidents[0].owner;
+  }
+
+  if (args.User) {
+      var user = args.User;
+  } else {
+      var res = executeCommand('getUsers', {current: true});
+      if (isError(res[0])) {
+          return res;
+      }
+      var user = res[0].Contents[0].id;
+  }
+
+  if (user !== owner || (user === owner && args.IgnoreOwner == "true")) {
+      var uri = '/investigation/' + invID + '/deleteuser/' + user;
+
+      var res = executeCommand('demisto-api-post', {uri: uri, body: body});
+      if (isError(res[0])) {
+          return res[0].Contents;
+      }
+
+      return 'Successfully removed ' + user + '.';
+  } else {
+      return 'User ' + user + ' is the owner.';
```

Comment:
Dont you want an error in that case?

Reply:
no. this can be run with the RunCommand on a lot of incidents (to clear left pane), so don't want to have errrors...